### PR TITLE
fixed travis hopefully

### PR DESCRIPTION
**Pull request by SunsetStationuser**
:cl: Changed yml to point to sunsetstation.dme and fixed the webhook
fix: Changed yml to point to sunsetstation.dme and fixed the webhook
/:cl:

[why]: # Hopefully this fixes travis

### CODE REVIEW

**Review by alexkar598 — DISMISSED** [review]
should be fine

File: sunset/tools/merge-upstream-pull-request.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ set -f # disable pathname expansion
 set -C # noclobber
 
 readonly BASE_BRANCH_NAME="upstream-merge-"
-readonly BASE_PULL_URL="https://api.github.com/repos/tgstation/tgstation/pulls"
+readonly BASE_PULL_URL="https://api.github.com/repos/SunsetStation/SunsetStation/pulls"
```
Comment (alexkar598) [inline sunset/tools/merge-upstream-pull-request.sh:16]: wait what the fuck @SunsetStationuser

File: sunset/tools/merge-upstream-pull-request.sh [hunk 2]
```diff
@@ -40,8 +40,8 @@ containsElement () {
 }
 
 # Make sure we have our upstream remote
-if ! git remote | grep tgstation > /dev/null; then
-   git remote add tgstation https://github.com/tgstation/tgstation.git
+if ! git remote | grep SunsetStation > /dev/null; then
+   git remote add SunsetStation https://github.com/SunsetStation/SunsetStation.git
```
Comment (alexkar598) [inline sunset/tools/merge-upstream-pull-request.sh:44]: same

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
not it

**Comment by alexkar598** [comment]
travis mean boi

**Comment by ghost** [comment]
not the cause of the issues.

**Comment by alexkar598** [comment]
github wont let us delete the branch for some reason....

**Pull request closed without merging**